docs/src/pages/[platform]/getting-started/accessibility/index.page.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: Accessibility
+metaTitle: Accessibility
+description: An introduction to building accessible apps with Amplify UI.
```

Comment:
Do we need to add a `metaDescription`? I saw that exist in other pages but not sure why we duplicate it with `description`

Reply:
description will also be used as the page meta description in the `<head>`, but it also outputs it in the UI (beneath the title in a larger font).

So, use metaDescription if you don't want it visible on the page, or use description if you want the same text in both places and visible on the page, or use both if for some reason you want the visible text to be different than the meta description.

Reply:
Got it, I am left behind in docs :(